src/pages/library/DataBrowser.js

```diff
@@ -47,21 +47,21 @@ const extractCatalogFilters = dataCatalog => {
     labels: _.values(datasetAccessTypes),
     labelRenderer: accessValue => {
       const lowerKey = _.toLower(accessValue)
-      const iconKey = accessValue === datasetAccessTypes.GRANTED ? 'unlock' : 'lock'
+      const iconKey = accessValue === datasetAccessTypes.Granted ? 'unlock' : 'lock'
```

Comment:
Is the accepted convention to use upper camel case for constant members like `Granted`? I would expect uppercase here.

I looked at https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide#naming-conventions but I don't think it's been updated with typescript concepts yet.

Reply:
So, I would usually say it should be all upper-case, but per conversations with @msilva-broad at the mob meeting, we changed it to this in order to use union types.